ansible/roles/ssc/templates/ssc-deploy.yml.j2

```diff
@@ -64,12 +64,24 @@ spec:
             value: "{{ ssc.denom }}"
           - name: GENESIS
             value: "{{ ssc_genesis }}"
+{% if ssc_genesis_url is defined %}
+          - name: GENESIS_URL
+            value: "{{ ssc_genesis_url }}"
+{% endif %}
           - name: LOGLEVEL
             value: "info"
           - name: PEERS
             value: "{{ ssc_peers }}"
+          - name: PERSISTENT_PEERS
```

Comment:
`PEERS` are already here. The same as previous comment. You can rename if you prefer, but no reason to duplicate.